CMakeLists.txt

```diff
@@ -598,9 +605,11 @@ if(BOUT_TESTS)
   add_subdirectory(tests/MMS EXCLUDE_FROM_ALL)
 
   # Targets for running the tests
-  add_custom_target(check-unit-tests
-    COMMAND ctest -R serial_tests --output-on-failure)
-  add_dependencies(check-unit-tests build-check-unit-tests)
+  if (BOUT_ENABLE_UNIT_TESTS)
```

Comment:
I don't see where the option for this is?

Reply:
It is set in the unit tests folder:
https://github.com/boutproject/BOUT-dev/pull/2596/files#diff-18b716ba95f75d9677945e5d932f2ec46a2c0e919a2123b154420422f94bd059